### ISSUE TITLE
Ignore linting in regex expressions

### ISSUE THREAD

**Issue by ryonakano** [vala-lang/vala-lint#141]
It seems like linting is ignored in strings:

```vala
// This code does not raise any error in vala-lint
var regex = new Regex ("^(xkb|m17n):(\\w+)");
```

However, if you use regex directly, vala-lint raise "space-before-paren" error:

```vala
// This code raise "space-before-paren" error in vala-lint
var regex = /^(xkb|m17n):(\w+)/;
```

You need to disable linting in the line to fix this, but I feel it's not smart:

```vala
var regex = /^(xkb|m17n):(\w+)/; // vala-lint=space-before-paren
```

It would be great if linting is disabled in regex expressions.
